octoprint_slic3r/profile.py

```diff
@@ -181,10 +182,9 @@ class SeamPositions(object):
     solid_infill_extruder = 1,
     support_material_contact_distance = 0.2,
     use_volumetric_e = 0,  				# 0 (unchecked) or 1 (checked)
-    threads = 2,
+    threads = min(multiprocessing.cpu_count() + 1, 16),
```

Comment:
This can be a good idea to achieve a better performance, but in my opinion this parameter must be overriden during profile import (indeed I would say it will be done unless something specifies not to override this setting)